src/compas_rhino/geometry/brep/brep.py

```diff
@@ -583,3 +584,55 @@ def split(self, cutter):
         """
         resulting_breps = self._brep.Split(cutter.native_brep, TOL.absolute)
         return [RhinoBrep.from_native(brep) for brep in resulting_breps]
+
+    def fillet(self, radius, edges=None):
+        """Fillet edges of the Brep.
+
+        Parameters
+        ----------
+        radius : float
+            The radius of the fillet.
+        edges : list(:class:`compas_rhino.geometry.RhinoBrepEdge`)
+            The edges to fillet.
+
+        Raises
+        -------
+        BrepFilletingError
+            If the fillet operation fails.
+
+        """
+        resulting_breps = self.filleted(radius, edges)
+        self._brep = resulting_breps.native_brep
+
+    def filleted(self, radius, edges=None):
+        """Returns a filleted copy of the Brep.
+
+        Parameters
+        ----------
+        radius : float
+            The radius of the fillet.
+        edges : list(:class:`compas_rhino.geometry.RhinoBrepEdge`)
+            List of edges to exclude from the operation. When None all edges are included.
+
+        Returns
+        -------
+        :class:`compas_rhino.geometry.RhinoBrep`
+            The resulting Brep.
+
+        Raises
+        -------
+        BrepFilletingError
+            If the fillet operation fails.
+
+        """
+        excluded_indices = [edge.native_edge.EdgeIndex for edge in edges] if edges else []
```

Comment:
sure this works? I think we saw together `BrepEdge.native_edge` was broken.

Reply:
Yes you are right. I just pushed the change.